crates/holochain_integrity_types/src/chain.rs

```diff
@@ -131,47 +146,82 @@ impl<H: Eq + Ord + std::hash::Hash> ChainFilter<H> {
         self
     }
 
+    /// Take all actions since the given timestamp.
+    /// If a timestamp was already set, the biggest of the two is kept.
```

Comment:
```suggestion
    /// Take all actions since the given timestamp.
    ///
    /// If a timestamp was already set, the biggest of the two is kept.
```

Minor thing but this affects the rendering in docs.rs. When you preview an item you see its first doc line and when you view the documentation for an item you see all of the content.

We have a huge number of doc comments that don't follow this style but it is the one recommended by the Rust documentation -> https://doc.rust-lang.org/rustdoc/how-to-write-documentation.html#documenting-components